galaxy_ng/tests/integration/api/test_collection_delete.py

```diff
@@ -0,0 +1,119 @@
+"""test_namespace_management.py - Test related to namespaces.
+
+See: https://issues.redhat.com/browse/AAH-1303
+
+"""
+import pytest
+import time
+
+from ..utils import get_client
+from ..utils import get_all_collections_by_repo
+from ..utils import get_all_repository_collection_versions
+
+
+pytestmark = pytest.mark.qa  # noqa: F821
+
+
+@pytest.mark.galaxyapi_smoke
+@pytest.mark.delete
+@pytest.mark.collection_delete
+def test_delete_collection(ansible_config, uncertifiedv2):
+    """Tests whether a colleciton can be deleted"""
+
+    config = ansible_config("ansible_partner")
+    api_client = get_client(
+        config=config,
+        request_token=True,
+        require_auth=True
+    )
+
+    # Enumerate published collection info ...
+    collectionv1 = uncertifiedv2[0]
+    cnamespace = collectionv1.namespace
+    cname = collectionv1.name
+    ckeys = []
+    for cv in uncertifiedv2:
+        ckeys.append((cv.namespace, cv.name, cv.version))
+
+    # Try deleting the whole collection ...
+    api_client(
+        f'/api/automation-hub/v3/collections/{cnamespace}/{cname}/',
+        method='DELETE'
+    )
+
+    # wait for the orphan_cleanup job to finish ...
+    time.sleep(5)
```

Comment:
does the delete response give you the task id in order to use `utils.wait_for_task`? ... that util may skip the sleep when tasks run fast, and could be improved to wait less